modules/iam/lambda/main.tf

```diff
@@ -46,7 +42,7 @@ data "aws_iam_policy_document" "lambda_nic_manager_policy" {
     ]
     condition {
       test     = "StringEquals"
-      values   = [data.aws_autoscaling_group.asg.name]
+      values   = [split("/", var.sensor_autoscaling_group_arn)[1]]
```

Comment:
Is the group name available using a more direct means instead of parsing the arn?

Reply:
Doesn't seem to be a cleaner way to do this ATM.